src/targets/registry.ts

```diff
@@ -362,7 +362,15 @@ export class RegistryTarget extends BaseTarget {
       );
     }
     // Update the manifest
-    const updatedManifest = { ...packageManifest, version };
+    const updatedManifest: {
+      version: string;
+      createdAt: string;
+      [key: string]: any;
+    } = {
+      ...packageManifest,
+      version,
+      createdAt: new Date().toISOString(),
```

Comment:
is there a plan to backfill this data?

any test that needs updating?

Reply:
Added a simple unit test to check if property is there
here is the PR to backfill the data for existing files
https://github.com/getsentry/sentry-release-registry/pull/148